packages/sources/cfbenchmarks/src/transport/utils.ts

```diff
@@ -1,15 +1,14 @@
+const LAST_UPDATE_ALLOWANCE_MS = 88200000 // 1 day + 30 mins in ms
```

Comment:
Why +30mins? Could we have a comment so we can remember why?

Reply:
added